dev/com.ibm.ws.microprofile.health.2.0_fat/fat/src/com/ibm/ws/microprofile/health20/fat/MultipleHealthCheckTest.java

```diff
@@ -4,11 +4,8 @@
  * are made available under the terms of the Eclipse Public License 2.0
  * which accompanies this distribution, and is available at
  * http://www.eclipse.org/legal/epl-2.0/
- * 
- * SPDX-License-Identifier: EPL-2.0
  *
- * Contributors:
```

Comment:
Are the contributors lines not required anymore?

Reply:
Yeah, with the EPL-2.0 update, there was discussion that the Contributors line wasn't needed anymore.